skaffold/src/main/kotlin/com/google/container/tools/skaffold/run/SkaffoldCommandLineState.kt

```diff
@@ -60,6 +62,12 @@ class SkaffoldCommandLineState(
             throw ExecutionException(message("skaffold.no.file.selected.error"))
         }
 
+        try {
+            AbstractSkaffoldRunConfiguration.checkConfiguration(environment.project)
```

Comment:
So if we put this into `SkaffoldExecutorService`, we have the following `SkaffoldExecutorService.instance.checkConfiguration`, with the added benefit of easier mocking and dynamic service lookup.

Reply:
thanks @ivanporty! @ZariaHoward and I discussed doing exactly this